scripts/dev-e2e.sh

```diff
@@ -153,31 +151,47 @@ main() {
     log "Armada is available"
   fi
 
-  echo "Creating $ARMADA_QUEUE queue..."
-  armadactl-retry create queue "$ARMADA_QUEUE"
+  armadactl-retry create queue "$ARMADA_QUEUE" 2>&1 | log_group "Creating $ARMADA_QUEUE queue"
 
-  echo "Loading Docker image $IMAGE_NAME into Armada cluster"
-  TMPDIR="$scripts/.tmp" "$AOHOME/bin/tooling/kind" load docker-image "$IMAGE_NAME" --name armada
+  (test -d "$scripts/.tmp" || mkdir "$scripts/.tmp";
+   TMPDIR="$scripts/.tmp" "$AOHOME/bin/tooling/kind" load docker-image "$IMAGE_NAME" --name armada 2>&1) \
+   | log_group "Loading Docker image $IMAGE_NAME into Armada cluster";
 
-  echo "Submitting SparkPI job"
-  PATH="$AOHOME/bin/tooling/:$PATH" "$scripts/submitSparkPi.sh"  2>&1 | tee submitSparkPi.log
+  # Pause to ensure that Armada cluster is fully ready to accept jobs; without this,
+  # proceeding immediately causes sporadic immediate job rejections by Armada
+  sleep 30
 
-  DRIVER_JOBID=$(grep '^Driver JobID:' submitSparkPi.log | awk '{print $3}')
-  EXECUTOR_JOBIDS=$(grep '^Executor JobID:' submitSparkPi.log | awk '{print $3}')
+  PATH="$scripts:$AOHOME/bin/tooling/:$PATH" JOBSET="$JOBSET" "$scripts/submitSparkPi.sh" 2>&1 | \
```

Comment:
why do you need to set the path here?

Reply:
`submitSparkPi.sh` uses `armadactl`, which is fetched earlier (see `fetch-armadctl()`) - although interactive devs often have it in `~/go/bin/armadactl`, it's absent on a fresh GH Actions container start.